ui/components/input/animate/_index.scss

```diff
@@ -7,24 +7,35 @@
  *
  * @name animate
  * @selector .slds-form-element__control-animated-label
- * @restrict .slds-form-element__control
+ * @restrict .slds-input
  * @variant
  */
 
 .slds-form-element__control-animated-label {
-	font-size: .75rem;
-
-	.slds-input {
-	  line-height: 1.5rem;
-	  min-height: calc(1.5rem + (#{$border-width-thin} * 2)); // For IE
-	}
+	font-size: 0.875rem;
+	position: relative;
 }
 
 
+.slds-input {
+	font-size: 0.875rem;
+    letter-spacing: 1px;
+    font-weight: 400;
+}
```

Comment:
I'm going to be pedantic now :) But can you please use consistent spacing for indentation. Notice how font-size is more indented than the rest.